source/features/minimize-upload-bar.css

```diff
@@ -1,9 +1,10 @@
 /* `.js-upload-markdown-image` excludes non-comment attachment drop areas. See #2086 */
-.rgh-minimize-upload-bar .js-upload-markdown-image.is-default .drag-and-drop {
+/* `:not(.js-comment-update)` excludes comment edition forms, as they lack the upload image button. See #4542 */
+.rgh-minimize-upload-bar form:not(.js-comment-update) .js-upload-markdown-image.is-default .drag-and-drop {
```

Comment:
> Since GitHub doesn't add an image upload button on edit forms, a solution is to just disable the feature there.

This is fine for the moment, we can revisit it later.

> The other solution would be to insert the missing button, but that would mean the feature isn't pure CSS anymore.

It is already a toggleable feature, I don't see any harm to make it more accessible via a little bit script.

> Or we could just drop the feature entirely, as I think it was mentioned somewhere.

It's https://github.com/refined-github/refined-github/issues/4960#issuecomment-955749784

But as I replied in that issue, I prefer the icon over the bar. The bar just looks out of place, and I'm aware of zero sites have this design, I've only seen it in a couple of component libraries.

Removing it would make a _huge_ visual difference which isn't worth it.